src/pages/balances/nfts.tsx

```diff
@@ -69,11 +98,13 @@ const NFTs: NextPage = () => {
       <NavTabs tabs={balancesNavItems} />
 
       <Box py={3}>
-        <Alert severity="info" sx={{ marginBottom: 6 }}>
+        <Alert severity="info">
           <AlertTitle>Use Safe Apps to view your NFT portfolio</AlertTitle>
           Get the most optimal experience with Safe Apps. View your collections, buy or sell NFTs, and more.
         </Alert>
 
+        <NftApps />
```

Comment:
I would suggest we memoize the component so it is not rerendered every time a new nft page is loaded.